### PR TITLE
Implemented new common menu

### PR DESCRIPTION
**Pull request by DanielBrierton**

### CODE REVIEW

File: templates/header.tpl [hunk 1]
```diff
@@ -22,6 +22,11 @@
 			user: JSON.parse('{{userJSON}}')
 		};
 	</script>
+	<script type="text/javascript">
+		window.cdMenuConfig = {
+			zenBase: 'http://localhost:8000'
```
Comment (Wardormeur) [inline templates/header.tpl]: reminder for tomorrow :)

File: templates/partials/cd-footer.tpl [hunk 2]
```diff
@@ -1,24 +1,48 @@
...
+      </svg>
+    </div>
+    <div class="col-sm-3">
+      <p class="cd-footer__address-line">CoderDojo Foundation,</p>
+      <p class="cd-footer__address-line">Dogpatch Labs Unit 1,</p>
+      <p class="cd-footer__address-line">The CHQ building,</p>
+      <p class="cd-footer__address-line">Custom House Quay,</p>
+      <p class="cd-footer__address-line">Dublin 1, Ireland</p>
+    </div>
+    <div class="col-sm-3"></div>
```
Comment (Wardormeur) [inline templates/partials/cd-footer.tpl]: col-sm-offset-3 :)

File: templates/partials/cd-header.tpl [hunk 3]
```diff
@@ -1,24 +1,257 @@
...
+      </div>
+      <ul class="cd-menu__content-block">
+        <li class="cd-menu__profile">
+          <span>
+            <img class="cd-menu__profile-pic" src="http://placekitten.com/32/32" />
+            <span class="cd-menu__profile-name">My Account</span>
+            <i class="cd-menu__sub-menu-icon fa fa-chevron-right"></i>
+          </span>
+          <div class="cd-menu__sliding-menu cd-menu__sliding-menu-secondary" data-toggle="closed">
+            <div class="cd-menu__close-row" style="background:white;">
```
Comment (Wardormeur) [inline templates/partials/cd-header.tpl]: inline style? :<

File: templates/partials/cd-header.tpl [hunk 4]
```diff
@@ -1,24 +1,257 @@
...
+        <button class="cd-menu__search-button" type="button"><span class="fa fa-search fa-2x"></span></button>
+      </div>
+      <div class="cd-menu__content-block cd-menu__account">
+        <a href="/register">Register</a>
+        <a href="/login">Login</a>
+      </div>
+      <ul class="cd-menu__content-block">
+        <li class="cd-menu__profile">
+          <span>
+            <img class="cd-menu__profile-pic" src="http://placekitten.com/32/32" />
```
Comment (Wardormeur) [inline templates/partials/cd-header.tpl]: placekitten as default? :p I'm up for it !

**Comment by DanielBrierton** [comment]
So, this is a little broken. Certain things appear even when you're not logged in. Don't merge until that's sorted.

**Comment by butlerx** [comment]
LGTM aside from issue in chat on mobile menu which is solved https://github.com/DanielBrierton/nodebb-theme-cd-persona/pull/1

**Pull request merged**